common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -700,3 +700,13 @@ d_darkspear = {
 		color3 = rgb { 255 231 167 }
```

Comment:
Sometimes I wonder why you don't expand the symbols to the max, to the borders.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123307588-ba440380-d533-11eb-8beb-84f051e0100a.png)
![image](https://user-images.githubusercontent.com/46384992/123307671-d3e54b00-d533-11eb-8a05-402ff91c7fa1.png)
![image](https://user-images.githubusercontent.com/46384992/123307739-e8c1de80-d533-11eb-810f-3a8c090256b2.png)
![image](https://user-images.githubusercontent.com/46384992/123307787-f24b4680-d533-11eb-89f2-46f75794befd.png)

</details>
P.S. There are many symbols like this.